cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -121,3 +121,4 @@ wildfly     openshift     10.0,10.1,11.0,12.0,13.0,14.0,15.0,16.0,17.0,18.0,19.0
 ----
 +
 The `TAGS` column represents the available image versions, for example, `10` represents the `rhoar-nodejs/nodejs-10` container image.
+If you want to learn more about the CLI commands, go to xref:../../cli_reference/developer_cli_odo/odo-cli-reference.adoc [odo CLI reference].
```

Comment:
S/If you want to learn more.../To learn more...

And for the link, give this a try: 
xref:../../cli_reference/developer_cli_odo/odo-cli-reference.adoc#odo-cli-reference[odo CLI reference]

Reply:
Thanks! I'll check it again and make sure the link works in a few minutes.

Reply:
Comments addressed, so I'm resolving this conversation.